examples/usb/device/bulk_trans/test.py

```
@@ -0,0 +1,72 @@
+    usb.util.release_interface(dev, intf)
+    usb.util.dispose_resources(dev)
+    print('Device closed successfully')
+
+except Exception as e:
+    print('Error:', str(e))
+    # Ensure device is closed on error
+    try:
+        usb.util.release_interface(dev, intf)
+        usb.util.dispose_resources(dev)
+        print('Device closed successfully')
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)